frontend/src/patientApp/timeOfTest/PatientLanding.tsx

```diff
@@ -19,11 +19,9 @@ const PatientLanding = () => {
               icon={"check-circle"}
               className="usa-media-block__img margin-right-2"
             />
-            <div className="usa-mmedia-block__body usa-prose">
+            <div className="usa-mmedia-block__body usa-prose margin-top-105">
               <h3 className="font-heading-sm">Questionnaire complete</h3>
-              <p className="margin-top-05">
-                You will receive a text notification when your result is ready.
-              </p>
+              <p className="margin-top-05"></p>
```

Comment:
This is a spacer gif. 😸  Is there a `margin-bottom-05` we can use?

Reply:
I know right! 😆 We actually don't need it at all. I only left it b/c we're temporarily cutting the text and this way I wouldn't have to rewrite the container. But happy to remove if we prefer.

Reply:
Commented out in 423ce75.